ChangeLog.d/add_full_no_platform_preset.txt

```diff
@@ -0,0 +1,3 @@
+Features
```

Comment:
I just realized: we do not consider `config.py` presets to be stable interfaces, so we must not mention them in the changelog.

Reply:
Oh, ok. I must admit I did wonder.